dash_app.py

```diff
@@ -0,0 +1,100 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import dash_table
+import more_itertools
+
+import plotly.express as px
+from covidactnow.datapublic.common_fields import CommonFields
+from dash.dependencies import Input, Output
+
+from libs import pipeline
+from libs.datasets import combined_datasets
+from libs.datasets.tail_filter import TagType
+from libs.datasets.timeseries import TagField
+
+
+external_stylesheets = ["https://codepen.io/chriddyp/pen/bWLwgP.css"]
+
+app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
+
+# Enable offline use.
+app.css.config.serve_locally = True
+app.scripts.config.serve_locally = True
+
+
+ds = combined_datasets.load_us_timeseries_dataset().get_subset(exclude_county_999=True)
```

Comment:
I realize this more of an example, but I'd like to include as many loading and app setup functions as possible in helpers functions - otherwise on any import (especially on imports in tests) it'll automatically trigger some expensive operations.  

Generally with flask apps, I have a `build_app` function that returns an instance of `app` and handles all of the initialization.  This makes it a bit tricky with the decorator here, even having all of the app building would be helpful to be in a function

Reply:
Good point. Moved the entire thing into a function. c48b5e49b9